inputs/linear_modes.pin

```diff
@@ -63,10 +63,10 @@ num_threads = 1         # maximum number of OMP threads
 <phoebus/mesh>
 #bc_vars = primitive
 
-<parthenon/meshblock>
-nx1 = 128
-nx2 = 128
-nx3 = 1
+# <parthenon/meshblock>
+# nx1 = 128
+# nx2 = 128
+# nx3 = 1
```

Comment:
What are these changes for?

Reply:
This sets the default to one meshblock which just made it easier for me to do the resolution study.